build/references/command-line-interface.md

```diff
@@ -655,3 +670,29 @@ Path to JSON file that defines aliases for Virtual Machine IDs. Defaults to `~/.
 
 The above example aliases the VM whose ID is `"tGas3T58KzdjLHhBDMnH2TvrddhqTji5iZAMZ3RXs2NLpSnhH"` to `"timestampvm"` and `"timerpc"`.
 
+### X-Chain / AVM <a id="avm-config"></a>
+
+This allows you to specify a config to be passed into the X-Chain. The default values for this config are:
+
+```javascript
+{
+  "index-transactions": false,
+  "index-allow-incomplete": false
+}
+```
+
+Default values are overridden only if explicitly specified in the config.
+
+The parameters are as follows:
+
+#### Transaction Indexing
```

Comment:
Does this indexing have the same behavior as the node-level indexer, where if the node ever runs with indexing disabled then it must always run in the future with `index-allow-incomplete`? If so, can you please write that here?

Reply:
Done